release_build_versions.txt

```diff
@@ -50,6 +50,8 @@ nebula latest
 
 nerdctl latest
 
+nomad 1.10.0
```

Comment:
Since we implement ` list_available_versions()` in `create.sh` we could additionally have
`nomad latest`
here to build the latest release automatically.